dawn/prototype/mylib_interface.hpp

```diff
@@ -21,6 +22,16 @@ inline decltype(auto) getCells(mylibTag, mylib::Grid const& m) { return m.faces(
 inline decltype(auto) getEdges(mylibTag, mylib::Grid const& m) { return m.edges(); }
 inline decltype(auto) getVertices(mylibTag, mylib::Grid const& m) { return m.vertices(); }
 
+// TODO Should be moved to a place common to all interfaces
```

Comment:
would that be `gtclang/src/gridtools/clang/interface.hpp`? I know it's a weird path, because that folder should not be in gtclang, but this is true for the whole folder.

Reply:
Probably, but I would introduce this new dependency in the mylib prototype. Maybe we should wait until we sorted out this problem?

Reply:
Ah, the dependency is already there.